frontend/src/metabase/components/CollapseSection/CollapseSection.unit.spec.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/display-name */
```

Comment:
🤔 Do we need this eslint disabler?

Reply:
Yeah, there is a functional component below in the header prop test and eslint would highlight it. Suppose we should maybe disable some rules for the test files